.github/workflows/testing-windows.yml

```diff
@@ -1,4 +1,4 @@
-name: CI Pipeline Tests Windows
+name: Tests Windows
```

Comment:
Suggestion to name them "Unit Tests Windows" to make it clear this will not run any integration or system tests.

Reply:
There are no integration or system tests on ipclib, so this makes no sense to do